lib/src/client.dart

```diff
@@ -43,7 +43,7 @@ import 'package:matrix/src/utils/try_get_push_rule.dart';
 
 typedef RoomSorter = int Function(Room a, Room b);
 
-enum LoginState { loggedIn, loggedOut }
+enum LoginState { loggedIn, loggedOut, softLoggedOut }
```

Comment:
I am pretty sure I wrote code somewhere, that assumes this only has 2 states, can we make it a minor bump and mention this as a breaking change?

Reply:
Sounds good to me.

I searched for all "LoginState" in the app so far and I think I covered everything in my soft logout PR there. When working with an enum we often use `switch` which let the analyzer ring when there is a new not handeld case

Reply:
https://github.com/famedly/matrix-dart-sdk/pull/1729